WebDriverAgentTests/IntegrationTests/FBAppiumTouchActionsIntegrationTests.m

```diff
@@ -211,7 +211,7 @@ - (void)testPress
   NSArray<NSDictionary<NSString *, id> *> *gesture =
   @[@{
       @"action": @"press",
-      @"options": @{
+      @"options": @{ // Here does not work in iOS 13
```

Comment:
pointer calculation in w3c?

- find_element => click it as `element.click` worked
- `@driver.action.click(e).perform` (w3c click like below) did not work (The hitpoint was wrong. Maybe it was portrait based point)

```
[debug] [XCUITest] Preprocessed actions: [
[debug] [XCUITest]   {
[debug] [XCUITest]     "type": "pointer",
[debug] [XCUITest]     "id": "mouse",
[debug] [XCUITest]     "actions": [
[debug] [XCUITest]       {
[debug] [XCUITest]         "type": "pointerMove",
[debug] [XCUITest]         "duration": 50,
[debug] [XCUITest]         "x": 0,
[debug] [XCUITest]         "y": 0,
[debug] [XCUITest]         "origin": {
[debug] [XCUITest]           "element-6066-11e4-a52e-4f735466cecf": "32000000-0000-0000-2AB1-000000000000"
[debug] [XCUITest]         }
[debug] [XCUITest]       },
[debug] [XCUITest]       {
[debug] [XCUITest]         "type": "pointerDown",
[debug] [XCUITest]         "button": 0
[debug] [XCUITest]       },
[debug] [XCUITest]       {
[debug] [XCUITest]         "type": "pointerUp",
[debug] [XCUITest]         "button": 0
[debug] [XCUITest]       }
[debug] [XCUITest]     ],
[debug] [XCUITest]     "parameters": {
[debug] [XCUITest]       "pointerType": "touch"
[debug] [XCUITest]     }
[debug] [XCUITest]   }
[debug] [XCUITest] ]
```